lib/perlsecret.pod

```diff
@@ -56,7 +56,7 @@ The Perl "I<secret operators>" have been discovered (or created) by
 Perl obfuscators and golfers, usually when looking for a shorter way to
 perform a given operation.
 Secret operators are not actually secret, and they are not actually
-operators either. The perl parser does not specifically recognise them,
+operators either. The perl parser does not specifically recognize them,
```

Comment:
I tend to use British spelling when I write English (they invented the language, after all). Although I might not be consistent, because most of the English I read is likely to be us-en anyway...

Reply:
Ok, fair enough! But then we should change `recognize` on line 62 below back to the British spelling instead?

Reply:
I have added a new commit with this change

Reply:
Actually, this was more of a comment. I'm not even sure I'm consistent with my use of British spelling. :smile: 

Thanks for your patches. I'm planning a new release soon, and I'll include them (with attribution, of course). Don't worry if I don't merge from github, though.